src/urban_disco/core.clj

```diff
@@ -1,74 +1,64 @@
 (ns urban-disco.core
   (:gen-class))
 
+(defn build-tile [x y] 
+  {:x        x
+   :y        y
+   :explored false})
 
+(defn build-row [width row-number]
+  (let [row (range width)]
+    (into [] (map #(build-tile % row-number) row))))
```

Comment:
`range` returns a list instead of a vector, you cannot use `update-in`/`assoc-in` on a list. 

There's still probably a better way to create this.